docs/recipes/ui/add-bulma.md

```diff
@@ -0,0 +1,32 @@
+# How do I add Bulma to a SAFE project?
+
+[Bulma](https://bulma.io/documentation/) is a free open-source UI framework based on flex-box that helps you create modern and responsive layouts.
+
+When using Feliz (the standard for a SAFE app), follow the instructions below. When using Fable.React, use the [Fulma](https://fulma.github.io/Fulma/) wrapper for Bulma.
+
+## 1. Add the Feliz.Bulma NuGet package to the solution
```

Comment:
I'm not sure that it's really correct to say that it's being added to the solution - the solution is a dotnet concept for gathering related projects. As far as I'm aware, Paket doesn't use the solution concept.

```suggestion
## 1. Add the Feliz.Bulma NuGet package to the client project
```